__tests__/articleRoutes.test.js

```diff
@@ -294,4 +294,15 @@ describe('Article Routes Test', () => {
         done();
       });
   });
+
+  it('should return trending articles when the endpoint is hit', (done) => {
+    chai.request(app)
+      .get(`${API_PREFIX}/trends`)
+      .end((err, res) => {
+        expect(res.status).to.be.eql(200);
+        expect(res.body).to.have.property('trends');
+        expect(res.body.trends).to.be.to.a('array');
+        done();
+      });
```

Comment:
I don't like tests written like this, and I'll keep saying it. Let your assertions be specific

Reply:
@Tunji17 have you seen this feedback? Please work on it so I can approve.

Reply:
Sorry, I didn't, @mazma1 there is no way I can assert what is in the array because it changes continuously, that's why I said it should have property trends so I know I am actually getting my array